herbert_core/classes/MPIFramework/parallel_worker.m

```diff
@@ -273,6 +273,9 @@
                 fbMPI = je.control_node_exch;
                 % is framework instance different from JE instance now?
                 intercomm= je.mess_framework;
+                % migrate job folder for message exchange without deleting the old
+                % one
+                je.migrate_job_folder(false);
```

Comment:
why is the folder migrated, but not cleaned up in this use-case?

Reply:
ibid (see above)